src/SprykerEco/Zed/UnzerApi/UnzerApiConfig.php

```diff
@@ -42,6 +42,16 @@ public function getUnzerApiCreateBasket(): string
         return $this->get(UnzerApiConstants::BASKET_RESOURCE_URL);
     }
 
+    /**
+     * @api
+     *
+     * @return string
+     */
+    public function getUnzerApiCreateMarketplaceBasket(): string
+    {
+        return $this->get(UnzerApiConstants::MARKETPLACE_BASKET_RESOURCE_URL);
```

Comment:
Is exception expected for cases when URL is not provided in configuration files?

Reply:
Do we need a default value or its major release?

Reply:
initial release